Pod/Classes/Forms/UIView+Lookup.swift

```diff
@@ -16,39 +16,25 @@ extension UIView {
     /// - parameter includeNonInteractable: Return views that do not have user interaction enabled. The default is false.
     ///
     /// - returns: An array of views, of type T, sorted by y position.
-    public func lookupSortedViews<T: UIView>(includeHidden: Bool = false, includeNonInteractable: Bool = false) -> [T] {
+    public func lookupSortedViews<T: UIView>(_ includeHidden: Bool = false, includeNonInteractable: Bool = false) -> [T] {
```

Comment:
These should be flagged with `@nonobjc final`. Double check the rest of the methods, any extension should have `@objc(rz_methodName` or `@nonobjc final`